packages/core/src/components/DataTable/TableHeader.tsx

```diff
@@ -11,10 +11,10 @@ export type Props = {
   editable?: boolean;
   /** Determines which set of header buttons to render. */
   editMode: boolean;
-  /** Without instantEdit the header renders Cancel and Apply, otherwise it just renders Done. */
-  instantEdit?: boolean;
   /** Height of the TableHeader, falls back to RowHeight if not specified. */
   height: number;
+  /** If instantEdit is disabled, header will render Cancel and Apply buttons during edit mode. */
```

Comment:
this description tripped me up a little - thinking it was `true` by default. how about `Renders a Done button. Defaults to Cancel and Apply buttons.`

Reply:
Oh yeah, `instantEdit` does default to `true`, I like your description more though, I'll go ahead and update to that

Reply:
where does it default to true? i don't see that being set in this file